opm/common/utility/numeric/GeometryUtil.hpp

```diff
@@ -0,0 +1,116 @@
+#ifndef GEOMETRYUTIL_H
+#define GEOMETRYUTIL_H
+
+#include <vector>
+#include <opm/common/utility/numeric/VectorUtil.hpp>
+#include <numeric>
+#include <cmath> 
```

Comment:
You should also `#include <algorithm>` for `min_element()`, `max_element()`, and `accumulate()`.

Reply:
I will add it, it got lost when I was merging my branch locally.

Reply:
take a look now.

Reply:
> take a look now.

Still not there.  Did you push your latest update?